<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpSender</class>
 <widget class="QWidget" name="TcpSender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="hostlabel">
       <property name="text">
        <string>主机：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hostEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="portlabel">
       <property name="text">
        <string>端口：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="clientBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statuslabel">
     <property name="text">
      <string>状态：等待打开文件</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
